chemprop/cli/hpopt.py

```diff
@@ -353,7 +359,12 @@ def main(args: Namespace):
     )
     val_loader = MolGraphDataLoader(val_dset, args.batch_size, args.num_workers, shuffle=False)
 
-    torch.manual_seed(args.pytorch_seed)
+    if args.pytorch_seed is None:
+        seed = torch.seed()
+    else:
+        seed = args.pytorch_seed
```

Comment:
Similar to Shih-Cheng's other comment, we could change this to
seed= PyTorch seed if PyTorch seed is not None else torch.seed()

I prefer this as it is less lines

Reply:
I have made this change.